test/e2e/traceflow_test.go

```diff
@@ -2557,7 +2559,7 @@ func runTestTraceflow(t *testing.T, data *TestData, tc testcase) {
 			pktCap.TransportHeader.ICMP = &v1beta1.ICMPEchoRequestHeader{}
 		}
 		if !reflect.DeepEqual(tc.expectedPktCap, pktCap) {
-			t.Fatalf("Captured packet should be: %+v, but got: %+v", tc.expectedPktCap, tf.Status.CapturedPacket)
+			t.Fatalf("Captured packet should be: %# v, but got: %# v", pretty.Formatter(tc.expectedPktCap), pretty.Formatter(tf.Status.CapturedPacket))
```

Comment:
I don't get this, Looks like it already prints the expected and captured packets before, how would a pretty format logs help here?

Reply:
the origin code will only print the pointer adress of inner-structure in the packet header, make it not easy to compare the actual conent in the header. the new package can print all the nested field in the packet.